Dockerfile

```diff
@@ -1,26 +1,41 @@
 FROM python:3.7-slim-buster
 
-ARG connectors=all
-COPY . /app
+ARG connectors=default
 
 RUN apt-get -qq update \
     && apt-get -qqy --no-install-recommends install \
         apt-utils \
         alien \
         libaio1 \
-        mongo-tools \
         mbuffer \
         wget \
+        git \
```

Comment:
why is this needed?

Reply:
It was needed to install `tap-mongodb` from `master` branch. It should go away as soon as you release updated version with `srv` support.